draft-ietf-tls-exported-authenticator.md

```diff
@@ -78,8 +78,14 @@ higher-layer protocol.  For example, multiplexed connection protocols
 like HTTP/2 {{!RFC7540}} do not have a notion of which TLS record
 a given message is a part of. 
 
+Exported Authenticators is meant to be used as a building block for
+application protocols.  Mechanisms such as those required to advertise
+support and handle authentication errors are not handled at the TLS layer.
+
 For simplicity, the mechanisms described in this document require a
-TLS version 1.2 or later.
+TLS version 1.2 or later.  It may be possible to adapt this mechanism
```

Comment:
What sequence of events lead to this conclusion regarding DTLS?  I would have thought that this works perfectly - nothing in this assumes that the authenticator is reliably carried (or even that the application attempts to transmit it), so it should work fine for DTLS.

The reason I am pushing on this is that we might want to use this for QUIC, which is functionally closer to DTLS in this regard and I just can't see how this would fail to work in that context.

If you want to get fancy, you might say that the application is responsible for providing support for transport of the authenticators and requests, which are likely to require fragmentation and reassembly.  Also, if the application depends on the reliable transmission of these messages, it has to provide a means of loss recovery.  That's not a lot of text.

If you keep this:
```suggestion
TLS version 1.2 or later.  It could be possible to adapt this mechanism
```

Reply:
I'll second @martinthomson's opinion here.  This is a local API -- the differences between TLS and DTLS are the application's problem and outside the scope of this document.

Reply:
This separation was created because exporters are not explicitly defined for DTLS. Considering EKR's point on the mailing list that exporters are already used in DTLS in another protocol, I think this point is moot and the text can be changed to just say TLS (or DTLS) at the beginning.